docs/src/main/sphinx/release/release-409.md

```diff
@@ -0,0 +1,86 @@
+# Release 409 (3 Mar 2023)
+
+## General
+
+* Add support for dropping nested fields with a `DROP COLUMN` statement. ({issue}`15975`)
+* Improve output of `EXPLAIN` queries to show statistics when a query contains
+  aggregations. ({issue}`16201`)
+* Improve performance of queries with distinct aggregations using table
```

Comment:
"distinct aggregations" is not a thing. Rephrase as "aggregations containing a `DISTINCT` clause"